### ISSUE TITLE
Wox has stopped suggesting applications

### ISSUE THREAD

**Issue by WolvenSpectre** [Wox-launcher/Wox#1762]
1) v 1.3.4.424 (auto update enabled)
2)Unknown.  I simply logged into my primary PC and while it had kept my custom black theme, all plugins were reset.  When I tried using it it would not suggest applications unless you added the .lnk or .exe to it.  All priority to most used apps seemed to be lost too.  For example, I would open my steamapps folder with it sometimes and that would show higher than other folders, but not anymore.

I have attempted the following:
a) Tried resetting folders in the settings
b) Waiting a half hour to reindex the PC.
c) Restarting the PC
d) reinstalling the software without uninstalling and uninstalling and reinstalling
e) Uninstalling using Revo Uninstaller on advanced, and reinstalling clean.
f) Restarting and Shutdown/booting from the clean install 

3)Microsoft Windows 10 Professional (x64) Build 15063.786 (1703/RS2)  
5)No such directory or file in AppData

6)Image of the issue demonstrated by trying to launch Notepad  https://www.dropbox.com/s/ruw76avd9kghu7p/Problem%20with%20WOX.png?dl=0

## Known problems for 1.3.183:

1. `System.NullReferenceException`: https://github.com/Wox-launcher/Wox/releases/tag/v1.3.424
2. `System.UriFormatException`: delete your old theme file
3. `System.Threading.Tasks.TaskCanceledException`: https://github.com/Wox-launcher/Wox/releases/tag/v1.3.424

## English

Try search existing issues before you open new one

1. Wox version / commit number: 
2. Steps to **reproduce** the bug:
3. Windows version:
4. Exception:
5. **whole** log file （`%APPDATA%\Wox\Logs\version\date.txt`):
6. Screenshot (optional): 

## 中文 / Chinese

提问之前搜索一下已经有的 issue

1. Wox 版本 / commit number：
2. 如何**重现** bug：
3. Windows 版本：
4. Exception：
5. **整个** log 文件 （`%APPDATA%\Wox\Logs\version\date.txt`）：
6. 截图 （可选）：

**Comment by lances101** [comment]
@WolvenSpectre As you can understand its hard for me to replicate what exactly happened on your machine. 

I need to log file in order to be able to understand what is going on on your machine.

Please follow the following steps: 
- clean the contents of `%appdata/Wox` and `%localappdata%/Wox`. I know you have done this previously, but please do.
- install Wox (if you had uninstalled it previously)
- launch Wox
- try searching for Notepad
- if nothing appears (which at this point is expected), go to `%appdata%/Wox/Logs%` and upload the latest log file.

**Comment by WolvenSpectre** [comment]
I rolled back to the version  1.3.183 and it is working again.  I am  
currently getting ready to travel for the holidays to see my family and  
probably won't get to this right away, but I will try this again when I  
can.  It may be in days or early January.

Until then thanks for the response and best wishes.

**Comment by lances101** [comment]
@WolvenSpectre, happy holidays. 

Once you have time lets try and figure out what is going on. Wox's Program plugin should not have any major problems in general, which is why your case is interesting.

**Comment by mkielar** [comment]
I've spotted Programs plugin to be failing when I add folders like `c:\windows\system32`. After that, indexing process fails, due to exceptions thrown by Program plugin. The exception says:

```
2018-01-04 22:33:24.9994|ERROR|App.RegisterAppDomainExceptions|-------------------------- Begin exception --------------------------
2018-01-04 22:33:24.9994|ERROR|App.RegisterAppDomainExceptions|First Chance Exception:
2018-01-04 22:33:25.0150|ERROR|App.RegisterAppDomainExceptions|Exception fulle name:
 <System.UnauthorizedAccessException>
2018-01-04 22:33:25.0306|ERROR|App.RegisterAppDomainExceptions|Exception message:
 <Odmowa dostępu do ścieżki „c:\Windows\System32\config\systemprofile\AppData\Local\Microsoft\Windows\INetCache\Content.IE5”.>
2018-01-04 22:33:25.0462|ERROR|App.RegisterAppDomainExceptions|Exception stack trace:
 <   w System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)>
2018-01-04 22:33:25.0619|ERROR|App.RegisterAppDomainExceptions|Exception source:
 <mscorlib>
2018-01-04 22:33:25.0619|ERROR|App.RegisterAppDomainExceptions|Exception target site:
 <Void WinIOError(Int32, System.String)>
2018-01-04 22:33:25.0775|ERROR|App.RegisterAppDomainExceptions|Exception HResult:
 <-2147024891>
2018-01-04 22:33:25.0775|ERROR|App.RegisterAppDomainExceptions|-------------------------- End exception --------------------------
```

The `Odmowa dostępu do ścieżki` part is Polish for `Access to path is denied`.

which would suggest it requires Administrator access to read through those folders inside `c:\windows\system32`. However, when I run Wox with admin rights, the issue still occurs. Needless to say, Program plugin fails to index files, and barely anything is being found from that point.

**Comment by lances101** [comment]
@mkielar that seems to be unrelated to @WolvenSpectre's issue. 
That error should've been closed with #1780. See https://github.com/Wox-launcher/Wox/issues/1779

**Comment by bao-qian** [comment]
Please try new release https://github.com/Wox-launcher/Wox/releases
If problem still exist, try reset by delete `%APPDATA%Wox` directory.
Reopen if above method won't help.
